### PR TITLE
added account resources diff app

### PR DESCRIPTION
**Pull request by vitalyg**

### CODE REVIEW

File: apps/account-resources-diff/app.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    return diffs
+
+def lambda_handler(event, context):
+    client = GraphQLClient("https://api.faros.ai/v0/graphql")
+    client.inject_token("Bearer " + event["farosToken"])
+
+    query = event["params"]["query"]
+    key = event["params"]["key"]
+    account1 = event["params"]["accountId1"]
```
Comment (mtth) [inline apps/account-resources-diff/app.py:38]: `from_account_id`/`to_account_id` or `old_account_id`/`new_account_id`?

File: apps/account-resources-diff/faros.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+appId: account-resources-diff
+displayName: Account Resources Difference
+description: >-
+  Compares the state of the infrastructure between two accounts and returns the
+  differences
```
Comment (mtth) [inline apps/account-resources-diff/faros.yaml:5]: Mention the expected parameters?

File: apps/account-resources-diff/app.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        item2 = list2[pointer2]
+
+        if item1[key] < item2[key]:
+            diffs.append(colored("- " + json.dumps(item1, indent=4, sort_keys=True), "red"))
+            pointer1 += 1
+        elif item1[key] > item2[key]:
+            diffs.append(colored("+ " + json.dumps(item2, indent=4, sort_keys=True), "green"))
+            pointer2 += 1
+        else:
+            diffs.append(json.dumps(item2, indent=4, sort_keys=True))
```
Comment (mtth) [inline apps/account-resources-diff/app.py:26]: Diff apps usually omit things that match, should we do the same? We could enable the current behavior with a parameter.

We should also document that diffing is only done on the key.

File: apps/iam-user-account-diff/app.py [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    
+    keys = {"": "userName",
+            ".attachedManagedPolicies": "policyArn",
+            ".groups.data": "groupName",
+            ".groups.data.attachedManagedPolicies": "policyArn",
+            ".groups.data.groupPolicyList": "policyName",
+            ".mfaDevices.data": "serialNumber",
+            ".permissionsBoundary": "permissionsBoundaryArn",
+            ".tags": "key",
+            ".userPolicyList": "policyName"}
```
Comment (mtth) [inline apps/iam-user-account-diff/app.py:198]: Much of the functionality here is generic and would be useful for other combinations of keys. I wonder if:

+ there is a better place for the diffing logic;
+ we can instrument our schema to automatically figure out these keys.

**Comment by chalenge** [comment]
@mtth @vitalyg I addressed the comments here and did some clean up. I will keep it here for now and migrate in another PR after this is merged: https://github.com/faros-ai/faros-python-client/pull/2 

Also created a separate ticket for apps migration

**Comment by tovbinm** [comment]
@chalenge let's finish this one up as well

**Comment by chalenge** [comment]
Closing this. Was merged with #15

**Pull request closed without merging**